tests/detection_finding_test.py

```diff
@@ -66,7 +66,6 @@ def test_detection_finding(self):
                 title="Title",
                 uid="123",
                 created_time=int(datetime.now().timestamp()),
-                created_time_dt=datetime.now(),
```

Comment:
Why are you removing this field?

Reply:
Added back, I believe this was accidental.